data/mods/Xedra_Evolved/items/inventor/armor.json

```diff
@@ -354,12 +385,22 @@
           "values": [
             { "value": "BONUS_DODGE", "add": 1 },
             { "value": "PERCEPTION", "add": 2 },
-            { "value": "MOTION_VISION_RANGE", "add": 30 }
+            { "value": "MOTION_VISION_RANGE", "add": 30 },
+            { "value": "RANGED_DAMAGE", "multiply": 0.33 },
+            { "value": "WEAPON_DISPERSION", "multiply": 0.33 }
           ]
         }
       ]
     },
-    "armor": [ { "encumbrance": 1, "coverage": 10, "covers": [ "eyes" ], "rigid_layer_only": true } ]
+    "armor": [
+      {
+        "encumbrance_modifiers": [ "NONE" ],
```

Comment:
With the error calling this line but saying it happens at line 421 I'm not sure if this is even an error or if the error is in a different part of the json.